packages/ListBox/src/components/Label/Label.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useListBox from "../../useListBox";
+import labelStyles from "./Label.styles";
+
+const propTypes = {
+  placeholder: PropTypes.string,
+};
+
+let lastKnownLabel = "";
+
+export default function Label(props) {
+  const { placeholder } = props;
+  const [state] = useListBox();
+  const { selectedOptions, isMulti, options } = state;
+
+  if (!isMulti && selectedOptions.length && state.options[selectedOptions[0]].preventDefaultOnSelect) {
+    if (!lastKnownLabel) return placeholder;
+
+    return lastKnownLabel;
+  }
+
+  function getListboxLabelForMulti() {
+    const optionsLength = selectedOptions.length;
+
+    const label = selectedOptions
+      .map(index => {
+        const option = options[index];
+        if (typeof option.content === "string") {
+          return option.content;
+        }
+
+        if (option.label) {
+          return option.label;
+        }
+
+        throw Error(
```

Comment:
```suggestion
        throw new Error(
```
Is supposed to work either way, but I think `new` is better to be explicit